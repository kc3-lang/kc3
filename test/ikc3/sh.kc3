quote require Sh
require Sh
quote Sh.escape("")
Sh.escape("")
quote Sh.escape("a")
Sh.escape("a")
quote Sh.escape("ab")
Sh.escape("ab")
quote Sh.escape("abc")
Sh.escape("abc")
quote Sh.escape(";")
Sh.escape(";")
quote Sh.escape("abc;")
Sh.escape("abc;")
quote Sh.escape("$abc")
Sh.escape("$abc")
quote Sh.escape("ab c")
Sh.escape("ab c")
quote Sh.escape("ab\nc")
Sh.escape("ab\nc")
