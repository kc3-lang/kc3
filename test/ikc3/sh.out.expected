require Sh
Sh
Sh.escape("")
"\"\""
Sh.escape("a")
"a"
Sh.escape("ab")
"ab"
Sh.escape("abc")
"abc"
Sh.escape(";")
"\";\""
Sh.escape("abc;")
"\"abc;\""
Sh.escape("$abc")
"\"\\$abc\""
Sh.escape("ab c")
"\"ab c\""
Sh.escape("ab\nc")
"\"ab\nc\""
